README.rst

```diff
@@ -34,8 +36,8 @@ for which we usually identify three common cases: **similarities, typos and vari
 `The first example notebook <https://skrub-data.github.io/stable/auto_examples/01_dirtycategories.html>`_
 goes in-depth on how to identify and deal with dirty data using the `skrub` library.
 
-What `skrub` does not
-~~~~~~~~~~~~~~~~~~~~~~~~~
+What `skrub` cannot do
```

Comment:
Maybe nitpick: I'm in favor of keeping "does not". It feels like the natural way of saying "We can't use skrub to do this". WDYT?